provisioner/plain-v0/README.md

```diff
@@ -164,32 +169,95 @@ metadata:
   name: combo
 spec:
   provisionerClassName: core.rukpak.io/plain-v0
-  bundleName: combo-0.0.1
+  bundleName: combo-v0.0.1
 ```
 
 Check the BundleInstance status to ensure that the installation was successful.
 
-```
+```bash
+$ kubectl get bundleinstance combo
 NAME    DESIRED BUNDLE   INSTALLED BUNDLE   INSTALL STATE           AGE
-combo   combo-0.0.1      combo-0.0.1        InstallationSucceeded   16h
+combo   combo-v0.0.1     combo-v0.0.1       InstallationSucceeded   38m
 ```
 
 From there, check out the combo operator deployment and ensure that the operator is present on the cluster.
 
 ```shell
-kubectl -n combo get deployments.apps combo-operator -o yaml 
+$ kubectl -n combo get deployments.apps combo-operator
+NAME             READY   UP-TO-DATE   AVAILABLE   AGE
+combo-operator   1/1     1            1           39m
+$ kubectl -n combo get deployments.apps combo-operator -o yaml | grep 'image:' | xargs
+image: quay.io/tflannag/combo:v0.0.1
 ```
 
-The operator should be successfully installed. To test out how the `plain-v0` provisioner continually reconciles
-BundleInstance resources, try deleting the combo deployment.
+The operator should be successfully installed.
+
+Next, the `plain-v0` provisioner continually reconciles BundleInstance resources. Let's try deleting the combo deployment:
 
 ```shell
 kubectl -n combo delete deployments.apps combo-operator
 ```
 
-Checking for the deployment again, it will be back on the cluster. The provisioner ensures that all resources required
+Check for the deployment again, it will be back on the cluster. The provisioner ensures that all resources required
 for the BundleInstance to run are accounted for on-cluster.
 
+### Upgrading the Combo Operator
+
+Let's say the combo operator released a new patch version, and we want to upgrade to that version.
+
+> Note: Currently upgrading a BundleInstance is a manual process that involves updating the spec.BundleName being referenced, and point to a new Bundle's metadata.Name. Automatically pivoting to a new Bundle is on the roadmap.
```

Comment:
It seems that we don't have an issue in GitHub to track this so users may get antsy for updates here. Should we create a task so that people coming to the repo can be aware of the status on this?